packages/backend/src/apps/toolbox/common/condition-is-true.ts

```diff
@@ -1,4 +1,29 @@
-import type { IJSONObject } from '@plumber/types'
+import type { IJSONObject, IJSONValue } from '@plumber/types'
+
+function compareNumbers(
+  field: IJSONValue,
+  condition: IJSONValue,
+  value: IJSONValue,
+): boolean {
+  // WARNING: can only compare safely up till Number.MAX_SAFE_INTEGER but BigInt cannot compare floats...
+  if (isNaN(Number(field)) || isNaN(Number(value))) {
```

Comment:
side note: actually, [number coercion](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Number#number_coercion) is a little scary in JS (e.g. `Number('')` === 0), we still have some unexpected behaviour to fix but out of scope of this PR.

Reply:
also leave this out until user feedbacks